app/Http/Controllers/ItemsController.php

```diff
@@ -70,8 +70,18 @@ public function unfaveItem(Request $request) {
     }
 
 
-    public function showUserHome() {
-        return view('welcome');
+    public function showUserHome(Request $request) {
+        $categoryCount = DB::table('items')
+          ->join('categories', 'items.category_id', '=', 'categories.id')
```

Comment:
Is there not a more model-y way to handle this? I am loathe to use `DB::` queries most of the time. Sometimes it's not avoidable, but...